feature-libs/asm/core/config/asm-config.ts

```diff
@@ -13,6 +13,18 @@ export abstract class AsmConfig extends OccConfig {
     customerSearch?: {
       maxResults?: number;
     };
+    userIdHttpHeader?: {
+      /**
+       * To fix certain features when a customer agent is emulating a user, we must send a header called
+       * "sap-commerce-cloud-user-id" to the backend fetching certain data as the user, rather than the
+       * agent. However, the header will break instances of Commerce Cloud who do not allow requests with
+       * this header. (For example, the configuration "corsfilter.commercewebservices.allowedHeaders" will
+       * need to allow it.)
+       *
+       * Enabling this feature will send the emulated user's IDs on requests that need it as context.
+       */
+      enable?: boolean;
```

Comment:
Is a default value set for this config?

Reply:
There's no default value; this is omitted. Omission has the same behavior, though I can add a default if it makes more sense to you.